README.md

```diff
@@ -67,6 +67,197 @@ We would appreciate any external contributions! 🙏
+* Style Transfer
+* Text Comparison
+* Geometry
+* Question Composition
+* Review Generation
+* Column Matching
+* Handling Commands
+* Author Identification
+* Order Generation
+* Language Identification
+* Prompt Generation
+* Text Span Selection
+* Hallucination
+* Title Generation
+* Document Classification
+* Relevancy Estimation
+* NER
+* Program Synthesis
+* Dialogue Prediction
+* Abuse Detection
+* Weblink Generation
+* Text Simplification
+* Reasoning: Numerical Reasoning
+* Reasoning: Entity Coreference
+* Reasoning: Reasoning on Actions
+* Reasoning: General Commonsense Reasoning
+* Reasoning: Abductive Reasoning
+* Reasoning: Pronoun Disambiguation
+* Reasoning: Temporal Reasoning
+* Reasoning: Multihop Reasoning
+* Reasoning: Reasoning on Social Interactions
+* Reasoning: Contextual Commonsense Reasoning
+* Reasoning: Numerical Commonsense Reasoning
+* Reasoning: Physical Reasoning
+* Reasoning: Logical Reasoning
+* Reasoning: Planning
+* Reasoning: Qualitative Reasoning
+* Reasoning: Deductive Reasoning
+* Reasoning: Causal Reasoning
+* Reasoning: Inverse-Causal Reasoning
+* Reasoning: Reasoning with Symbols
+
+Language
+* English
+* Hindi
+* Gujarati
+* Spanish
+* Farsi
+* Italian
+* Japanese
+* Polish
+* Oriya
+* Bengali
+* Malayalam
+* Persian
+* Kannada
+* Arabic
+* Finnish
+* Turkish
+* Chinese
+* German
+* Bulgarian
+* Swedish
+* Vietnamese
+* Thai
+* Burmese
+* Malay
+* Laotian
+* Khmer
+* Bahasa
+* Filipino
+* French
+* Somali
+* Tagalog
+* Igbo
+* Korean
+* Marathi
+* Punjabi
+* Telugu
+* Tamil
+
+Domain:
+* Movies
+* Art
+* Architecture
+* Geography
+* History 
+* Music
+* News
+* Society 
+* Law
+* Justice 
+* Anthropology
+* 9/11 reports
+* School science textbooks
+* Fiction
+* Personal narratives
+* National Football League (NFL) game summaries
+* Abstract algebra
+* Anatomy
+* Astronomy
+* Business ethics
+* Clinical knowledge
+* College biology
+* College chemistry
+* College computer science
+* College mathematics
+* College medicine
+* College physics
+* Computer security
+* Conceptual physics
+* Econometrics
+* Electrical engineering
+* Elementary mathematics
+* Formal logic
+* Global facts
+* High school biology
+* High school chemistry
+* High school computer science
+* High school european history
+* High school geography
+* High school government and politics
+* High school macroeconomics
+* High school mathematics
+* High school microeconomics
+* High school physics
+* High school psychology
+* High school statistics
+* High school us history
+* High school world history
+* Human aging
+* Human sexuality
+* International law
+* jurisprudence
+* Logical fallacies
+* Machine learning
+* Management
+* Marketing
+* Medical genetics
+* Miscellaneous
+* Moral disputes
+* Moral scenarios
+* Nutrition
+* Philosophy
+* Prehistory
+* Accounting
+* Law
+* Medicine
+* Psychology
+* Public relations
+* Security  studies
+* Sociology
+* Us foreign policy
+* Virology
+* World religions
+* Social Media
+
```

Comment:
How would you categorize a science paper? Say: 
 - A biology paper 
 - or an EMNLP paper